dependencies/softfloat/berkeley-softfloat-3/build/Linux-RISCV-GCC/Makefile

```diff
@@ -36,18 +36,16 @@
 
 SOURCE_DIR ?= ../../source
 SPECIALIZE_TYPE ?= RISCV
-MARCH ?= rv64gcv_zfh_zfhmin
-MABI ?= lp64d
-CC ?= riscv64-unknown-linux-gnu-gcc
+
```

Comment:
I think you can leave these as they were. Just delete them from the `$(CC)` line.